### PR TITLE
added 3 java programs

### LINKED ISSUE #270: want to add basic java programs
I want to add few basic java programs

Description:
Add programs on java for factorial, fibonacci and swaptwonumbers

**Languages**

-   [x] Java

### PR DESCRIPTION
**Pull request by AnOnYmOus001100**
# Added 3 java programs

Added factorial.java, fibonacci.java and swapToNumbers.java
factorial.java : finds the factorial of a number
fibonacci.java : prints fibonacci series upto n
swapToNumbers.java : swaps two numbers without using temoprary variable

Fixes: #270 

# Checklist:

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] My changes generates no new warnings
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] New and existing unit tests pass locally with my changes
-   [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by kaustubhgupta** [comment]
@AnOnYmOus001100 Can you specify which issue has been assigned to you?

**Comment by AnOnYmOus001100** [comment]
Here is the issue
want to add basic java programs #270

File: Java/factorial.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/**
```
Comment (kaustubhgupta) [inline Java/factorial.java:1]: Remove these doc strings from all files

File: Java/factorial.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	Author: AnOnYmOus001100
```
Comment (kaustubhgupta) [inline Java/factorial.java]: Remove extra spaces from programs

File: Java/factorial.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+	Date: 04/10/2020
```
Comment (kaustubhgupta) [inline Java/factorial.java]: Update the readme of Java folder

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by AnOnYmOus001100** [comment]
ok, done removing the docstrings. updated the codes

**Review by kaustubhgupta — APPROVED** [review]

**Pull request merged**